src/libcore/option.rs

```diff
@@ -762,12 +763,21 @@ impl<T> AsSlice<T> for Option<T> {
 
 #[stable]
 impl<T> Default for Option<T> {
-    #[stable]
     #[inline]
     #[stable]
     fn default() -> Option<T> { None }
 }
 
+#[stable]
+impl<T: fmt::String> fmt::String for Option<T> {
```

Comment:
I considered adding a `String` implementation for `Option`, where `Some` would print the string, and `None` would print an empty string. But it's also not hard to pop open the option, and might be less confusing?